ocs_ci/deployment/vmware.py

```diff
@@ -379,14 +380,23 @@ def post_destroy_checks(self):
         Post destroy checks on cluster
         """
         pool = config.ENV_DATA["cluster_name"]
-        if self.vsphere.is_resource_pool_exist(pool, self.datacenter, self.cluster):
+        try:
+            rp_exists = self.vsphere.is_resource_pool_exist(
+                pool, self.datacenter, self.cluster
+            )
+        except vmodl.fault.ManagedObjectNotFound as e:
+            if (
+                "has already been deleted or has not been completely created"
+                not in str(e)
+            ):
+                raise
+        if rp_exists:
```

Comment:
will not be initialized if execution goes to except

Reply:
Good catch